src/contract/table.ts

```diff
@@ -152,9 +158,7 @@ export class Table<RowType = any> {
      * @returns {Promise<TableRow>} Promise resolving to a single table row.
      */
     async get(value?: API.v1.TableIndexType | string, params: QueryParams = {}): Promise<RowType> {
-        const fieldToIndexMapping = this.getFieldToIndex()
-
-        const tableRowsParams = {
+        const tableRowsParams: any = {
```

Comment:
Can't the type just be inferred here?

Reply:
It was throwing an error when it was inferring it, which is why it's also using `any` here from a previous commit:

https://github.com/wharfkit/contract/blob/c40e1f8c194d82f730a039b74955b9884a3f084d/src/contract/table.ts#L112-L114

Reply:
This rings a bell, but I can't remember the specifics. What was the error again? I just think that there has to be a way to get this working without the `any` 😄

Reply:
The get_table_rows call starts throwing a type error because of the mix of parameters we're passing. 

It might be possible to remove the any if we more intelligently crafted different types of queries to pass in, based on the params, instead of just making one big set of params. Not sure though!